index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start toolbar/header -->
+    <header class="toolbar">
+      <div class="logo_placeholder">
+        <a href="index.html" class="logo">Welcome</a>
+      </div>
+      <nav class="menu">
+        <button type="button" class="btn">
+          <i class="material-icons">menu</i>
+        </button>
+      </nav>
+    </header>
+    <!-- end toolbar/header -->
+
+    <!-- start main -->
+    <section class="headline background">
+      <h1>
+        <span>Hey there. I’m Radmila</span>
+        <span class="green">I’m a software developer</span>
+      </h1>
```

Comment:
- [ ] I see that you have tried to implement the previously required change here. Instead of using a `span` and wrapping it in a `h1` tag, wrap the section header/main title  `Hey there. I'm Radmila` in a `h1` tag and the subtitle `I'm a software developer` in a `hx` tag.  As the previous code reviewer mentioned about semantic HTML tags, they help improve accessibility for people who can't easily read screens.

Reply:
I chose the span tag because the h1 tag is a flex item,  by adding another child in this flex section, the space between elements would change; if I wrap h1 and hx in a div, then the section wouldn't have a direct child h1 element.  How do you suggest I proceed?